src/SFA.DAS.Authorization/Cache/AuthorizationCacheService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace SFA.DAS.Authorization.Cache
+{
+    public class AuthorizationCacheService : IAuthorizationCacheService
```

Comment:
Is this class needed? i.e. Could its implementation be moved to `AuthorizationHandlerCacheDecorator` instead?

Reply:
Feels like it is better to keep separate simply because they have different responsibilities. The service is usable elsewhere if required and the testing is easier.